cpp/mysql_diff_schemas.cc

```diff
@@ -53,16 +53,59 @@ inline bool SchemaLineIsLessThan(const std::string &line1, const std::string &li
 }
 
 
+std::string ExtractNextBacktickQuotedText(const std::string::const_iterator &end, std::string::const_iterator &cp) {
+    // Find the opening backtick:
+    while (cp != end and *cp != '`')
+        ++cp;
+    if (unlikely(cp == end))
+        return "";
+    ++cp; // Skip over the opening backtick.
+
+    std::string quoted_text;
+    while (cp != end and *cp != '`')
+        quoted_text += *cp++;
+    if (unlikely(cp == end))
+        return "";
+    ++cp; // Skip over the closing backtick.
+
+    return quoted_text;
+}
+
+
+void ExtractTriggerNameAndTable(const std::string &trigger, std::string * const name, std::string * const table) {
+    auto cp(trigger.cbegin());
+    *name = ExtractNextBacktickQuotedText(trigger.cend(), cp);
```

Comment:
I don't think it is as simple as that => optional DEFINER statement could use backticks as well

Reply:
In our input the DEFINER part of the statement is now always present and will be skipped over,